src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -1144,4 +1165,30 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn test_decimal_divide() {
+        let cases = vec![
+            (str2dec("2.2"), str2dec("1.1"), str2dec("2.0")),
+            (str2dec("12.3"), str2dec("-0.3"), str2dec("-41")),
+            (str2dec("12.3"), str2dec("0.3"), str2dec("41")),
+            (str2dec("12.3"), str2dec("0"), None),
+            (None, str2dec("2"), None),
+            (str2dec("123"), None, None),
+        ];
```

Comment:
```
            (
                ScalarFuncSig::DivideDecimal,
                str2dec("12.3"),
                str2dec("0.0"),
            ),
            (
                ScalarFuncSig::DivideDecimal,
                str2dec("12.3"),
                str2dec("-0.0"),
            ),
```
I notice there are another 2 tests in builtin_arithmetic.rs 195-211 lines
Would you mind and these tests here?

Reply:
I covered these cases by:
```rust
(str2dec("12.3"), str2dec("0"), None),
```